rook/run_query/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -eux
+
+# export the lib path
+export LD_LIBRARY_PATH=/usr/lib64/ceph
+
+yum install -y bc
```

Comment:
another alternative is to use `awk`:

```bash
echo 1.2 3.4 | awk '{print $1 + $2}'
```

it is usually preferable to avoid running `yum` as part of the workflow, to guard against transient connectivity issues. Plus this might be executed multiple times

Reply:
Thanks. Done